plugins/node/opentelemetry-plugin-dns/test/integrations/dnspromise-lookup.test.ts

```diff
@@ -155,7 +155,7 @@ describe('dns.promises.lookup()', () => {
             // tslint:disable-next-line:no-any
             hostname: hostname as any,
             forceStatus: {
-              code: CanonicalCode.INVALID_ARGUMENT,
+              code: StatusCode.ERROR,
```

Comment:
There is an `if` here where both the `if` and `else` have the same code. this if can be cleaned:
```js
if (semver.lt(process.versions.node, '9.0.0'))
```